src/calculateRentalCost.js

```diff
@@ -3,8 +3,19 @@
  *
  * @return {number}
  */
+const discountsDays1 = 3;
+const discountsDays2 = 7;
+const discount1 = 20;
+const discount2 = 50;
```

Comment:
The names of the constants `discountsDays1`, `discountsDays2`, `discount1`, and `discount2` could be more descriptive. Consider renaming them to reflect their purpose more clearly, such as `MIN_DAYS_FOR_DISCOUNT`, `MIN_DAYS_FOR_MAX_DISCOUNT`, `SMALL_DISCOUNT_AMOUNT`, and `LARGE_DISCOUNT_AMOUNT` respectively.